org.osgi.test.assertj.cm/src/main/java/org/osgi/test/assertj/cm/configuration/ConfigurationConditions.java

```diff
@@ -0,0 +1,115 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+
+package org.osgi.test.assertj.cm.configuration;
+
+import java.util.Objects;
+
+import org.assertj.core.api.Condition;
+import org.assertj.core.condition.VerboseCondition;
+import org.osgi.service.cm.Configuration;
+
+/**
+ * A Utility-Class thats Provides public static methods to create
+ * {@link Condition}s for an <code>Configuration</code>}
+ */
+public class ConfigurationConditions {
```

Comment:
Should this be default visibility or is it user facing?

Reply:
The same applies to all the methods

Reply:
good questions.

it could be also be helpfull for users to Conditions especially on lists

https://joel-costigliola.github.io/assertj/assertj-core-conditions.html